SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -646,6 +646,15 @@ void GenerateActorSchema(FComponentIdGenerator& IdGenerator, UClass* Class, TSha
 
 	ActorClassPathToSchema.Add(Class->GetPathName(), ActorSchemaData);
 
+	// Cache the NCD for this Actor
+	if (AActor* CDO = Class->GetDefaultObject<AActor>())
+	{
+		if (NetCullDistanceToComponentId.Find(CDO->NetCullDistanceSquared) == nullptr)
```

Comment:
Will these have stable bit patterns? For instance adding 10 and 10+epsilon may result in 2 entries with the same distance

Reply:
NetCullDistances aren't calculated/evaulated, they're just set, so there shouldn't be floating point errors.